lib/audited/auditor.rb

```diff
@@ -51,7 +51,7 @@ def audited(options = {})
         self.audit_associated_with = audited_options[:associated_with]
 
         if audited_options[:comment_required]
-          validates_presence_of :audit_comment, if: :auditing_enabled
+          validate :presence_of_audit_comment
           before_destroy :require_comment
```

Comment:
How about adding a condition to the callback here to `if audited_options[:on].include?(:destroy)` instead of the added check inside the function?

Reply:
The original logic behind doing it the way I did is because I didn't want this to be skipped via `if audited_options[:on].include?(:destroy)` if you weren't utilizing the `on:` option at all (i.e. I was assuming `on:` would be empty if you weren't utilizing it), but now that I'm aware of the `normalize_audited_options` function, definitely makes more sense to have the condition directly on the callback. 😄